src/components/layouts/php-collection-page-layout.tsx

```diff
@@ -0,0 +1,1161 @@
+                                      >
+                                        <div className="flex items-center py-2 leading-tight">
+                                          <div className="flex items-center flex-grow mr-2">
+                                            <small className="w-4 pt-px font-normal text-gray-500 scale-75 dark:text-gray-600 font-xs">
+                                              {isComplete ? `✔️` : index + 1}
+                                            </small>
+                                            <PlayIcon className="mx-1 text-gray-500 dark:text-gray-100" />
+                                          </div>
+                                          {lesson.path && (
+                                            <Link href={lesson.path}>
+                                              <a
+                                                onClick={() => {
+                                                  track(
+                                                    `clicked collection video link on course page`,
+                                                    {
+                                                      course: course.slug,
+                                                      video: lesson.slug,
+                                                      collection: playlist.slug,
+                                                    },
+                                                  )
+                                                }}
+                                                className="flex items-center w-full hover:underline"
+                                              >
+                                                <Markdown className="mt-0 prose text-gray-700 dark:prose-dark md:dark:prose-lg-dark md:prose-lg dark:text-gray-100">
+                                                  {lesson.title}
+                                                </Markdown>
+                                              </a>
+                                            </Link>
+                                          )}
+                                        </div>
+                                      </li>
+                                    )
+                                  },
+                                )}
+                              </ul>
+                            </div>
+                          </li>
+                        )
+                      })}
+                    </ul>
+                  </div>
+                )}
+              </section>
+              {/*Start of lessons block*/}
+              <div>
+                <ModulesList
+                  modules={lessonModules}
+                  render={(lessonList) => (
+                    <LessonsList
+                      lessons={lessonList}
+                      completedLessonSlugs={completedLessonSlugs}
+                    />
+                  )}
+                />
```

Comment:
render props felt appropriate here as the module list shouldn't care how lessons are being rendered, just where in the component they are.

This also avoids having to pass `completedLessonSlugs` to `ModulesList` which isn't a concern of the component